modules/woocommerce-analytics/classes/wp-woocommerce-analytics-universal.php

```diff
@@ -76,37 +76,84 @@ public function enqueue_tracking_script() {
 		wp_enqueue_script( 'woocommerce-analytics', esc_url( $filename ), array(), null, false );
 	}
 
+	/**
+	 * Default event properties which should be included with all events.
+	 *
+	 * @return array Array of standard event props.
+	 */
+	public function get_common_properties() {
+		return array(
+			'blog_id'     => Jetpack::get_option( 'id' ),
+			'ui'          => $this->get_user_id(),
+			'url'         => home_url(),
+			'woo_version' => WC()->version,
```

Comment:
How about keeping this inline with other property names: `wv`?

Reply:
Prefer clarity over terseness here. See also @deltaWhiskey comment in support of clearer props names: https://github.com/Automattic/jetpack/pull/15028#discussion_r395233504